store/mockstore/unistore/cophandler/mpp_exec.go

```diff
@@ -153,16 +155,44 @@ func (e *exchSenderExec) next() (*chunk.Chunk, error) {
 			}
 			return nil, nil
 		} else if chk != nil {
-			for _, tunnel := range e.tunnels {
-				tipbChunks, err := e.toTiPBChunk(chk)
-				if err != nil {
-					for _, tunnel := range e.tunnels {
-						tunnel.ErrCh <- err
+			if e.exchangeTp == tipb.ExchangeType_Hash {
+				rows := chk.NumRows()
+				targetChunks := make([]*chunk.Chunk, 0)
```

Comment:
Make with cap by using length of targetChunks

Reply:
fixed